### PR TITLE
Capture birth place correctly

### PR DESCRIPTION
**Pull request by ondenman**
MemberPage is not currently capturing the birth place of some members correctly. 

For example:
```yaml
:url: https://www.parlament.al/deputet/gjovalin-bzhetaj/
:to_h:
  :birth_date: '1967-08-21'
  :birth_place: '21.08.1967 Vendlindja: Shkodër'
  :area: ''
  :commissions: 'Grupi parlamentar: PDKomisioni parlamentar: Komisioni për Edukimin
    dhe Mjetet e Informimit Publik, anëtar.'
```

This PR ensures that the birth_place field captures the birth place only.

### CODE REVIEW

**Comment by ondenman** [comment]
@tmtmtmtm https://github.com/everypolitician-scrapers/albania-kuvendi/pull/13 extracts the MemberPage class but it is not capturing all data correctly. This PR fixes the `birth_place` field. It is currently failing `ensure-regression-tests` with the error: ```The command "bundle exec rake" exited with 0.
0.14s$ bash <(curl -fsSL https://github.com/everypolitician/ensure-regression-tests/raw/v0.1.0/ensure-regression-tests)
error: pathspec 'extract-member-page-class' did not match any file(s) known to git.```

File: lib/member_page.rb [hunk 1]
```diff
@@ -8,7 +8,8 @@ class MemberPage < Scraped::HTML
   end
 
   field :birth_place do
-    cells.find(-> { ':' }) { |t| t.include?('Vendlindja') }.split(':', 2).last.tidy.sub(/\.$/, '')
+    return nil unless birth_place_record
```
Comment (tmtmtmtm) [inline lib/member_page.rb]: No need for an explicit `nil` there.

File: test/data/gjovalin-bzhetaj.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+:class: MemberPage
+:url: https://www.parlament.al/deputet/gjovalin-bzhetaj/
+:to_h:
+  :birth_date: '1967-08-21'
+  :birth_place: 'Shkodër'
+  :area: ''
+  # TODO: This is capturig both parliamentary groups and commissions (#14)
```
Comment (tmtmtmtm) [inline test/data/gjovalin-bzhetaj.yml]: typo 'capturig'

**Comment by tmtmtmtm** [comment]
I've changed this to merge into `master` rather than #13

**Pull request merged**